submissions/akinbobolaFaith/css/contactForm.css

```diff
@@ -8,8 +8,8 @@ body {
     box-sizing: border-box;
 }
 
-nav a[href*="*"] {
-     font-size: 31px;
+nav a[href="#"] {
```

Comment:
kindly give this a class name. Imagine you now add a link, this styling will be affected and thus you need to update it again. Same goes for others